packages/node-plop/src/node-plop.js

```diff
@@ -100,6 +100,8 @@ async function nodePlop(plopfilePath = "", plopCfg = {}) {
       targets = [targets];
     }
     const config = Object.assign(
+      {}, // forces shallow copy of plopCfg
```

Comment:
This was not part of the suggested fix, but I felt like it would be better to at least shallow copy, than to merge directly into the original object.

Is this needed?

Reply:
This is absolutely a good addition, great fix!